test/e2e-kind.sh

```diff
@@ -133,3 +133,13 @@ go test -race -count=1 -timeout=5m -tags=e2e ./test/proxyprotocol/... \
   --ingressendpoint="${IPS[0]}" \
   --ingressClass=kourier.ingress.networking.knative.dev \
   --cluster-suffix="$CLUSTER_SUFFIX"
+
+echo ">> Set IdleTimeout to 50s"
+kubectl -n "${KOURIER_CONTROL_NAMESPACE}" patch configmap/config-kourier --type merge -p '{"data":{"stream-idle-timeout":"50s"}}'
+kubectl -n "${KOURIER_CONTROL_NAMESPACE}" rollout restart deployment/net-kourier-controller
+
+echo ">> Running IdleTimeout tests"
+go test -v  -tags=e2e ./test/e2e -run "^TestIdleTimeout*" \
```

Comment:
Could you move the test into `test/e2e/timeout/` and run the test as `go test -v -tags=e2e ./test/e2e/timeout` ?